libs/content-search-toolkit/src/queries/search.ts

```diff
@@ -34,11 +34,18 @@ export const searchQuery = (
   let minimumShouldMatch = 1
 
   should.push({
-    simple_query_string: {
+    multi_match: {
+      fields: [
+        'title^6', // note boosting ..
+        'title.stemmed^2', // note boosting ..
```

Comment:
Not sure what you mean by note boosting ..

Do you mean that the comment is a note implying that the change is boosting results?

Or is this a leftover comment? I'm just wondering :D

Reply:
example the "^6" part of "title^6" is giving the fields different weight/boosting ... just added the comments so the reader will note that

Reply:
Ok cool 👍